cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -18,6 +18,8 @@
 #include "gandiva/gdv_function_stubs.h"
 
 #include <utf8proc.h>
+#include <jsoncons/json.hpp>
```

Comment:
Don't know if it is okay to use hpp references in Gandiva files. 
Normally, I see cases in Gandiva where a wrapper .h file is created to avoid having these .hpp references (something related to compatibility problems while using llvm).

You may take a look at the **decimal_xlarge.cc** file which is a wrapper to use the boost .hpp file to handle some decimal operations